src/async_fuse/mod.rs

```diff
@@ -42,7 +45,7 @@ pub async fn start_async_fuse(
         StorageParams::S3(_) => {
             let fs: memfs::MemFs<memfs::S3MetaData<S3BackEndImpl>> = memfs::MemFs::new(
                 &args.mount_dir,
-                args.storage_config.cache_capacity,
+                CACHE_DEFAULT_CAPACITY,
```

Comment:
Do we still have a way to set the cache capacity? If yes, why do we pass the default capacity in here?

Reply:
This capacity is used by `GlobalCache`, which will be removed soon. And I removed the related config, so there are not direct ways to set it. If we still need to set the capacity for it, I can try to use the new capacity config (in blocks) defined for the new `MemoryCache`, to handle this temporarily.